blockprotocol/blockprotocol
# Redirect logged-in users visiting root to `/dashboard`

## 🌟 What is the purpose of this PR?

- Redirects logged-in users to `/dashboard` from `/` via [Next.js Middleware](https://nextjs.org/docs/advanced-features/middleware)


## 🔗 Related links
[Asana task](https://app.asana.com/0/1200211978612931/1202765662321696/f) (internal)


## ❓ How to test this?

1.  Checkout the branch / view the deployment
1.  Login
2. Try opening root `/` . You should be redirected to `dashboard`
3. Logout
4. Try opening `/`. You should be able to see homepage
